conf/application.conf

```diff
@@ -133,7 +139,7 @@ tracking-consent-frontend {
 }
 
 features {
-  welsh-translation = true
+  welsh-translation = false
```

Comment:
Why are we disabling Welsh translation?

Reply:
Because similarly to registration, the returns service will go into private beta with welsh translation disabled - it will only be enabled for public beta